ros2_control_demo_bringup/launch/rrbot.launch.py

```diff
@@ -80,7 +80,7 @@ def generate_launch_description():
     robot_controller_spawner = Node(
         package="controller_manager",
         executable="spawner.py",
-        arguments=["forward_position_controller", "-c", "/controller_manager"],
+        arguments=["joint_trajectory_controller", "--controller-manager", "/controller_manager"],
```

Comment:
There is no need to change this line. We actually want to show here how one can switch controllers. Therefore, default started controller is a forwarding controller.

```suggestion
        arguments=["forward_position_controller", "--controller-manager", "/controller_manager"],
```

Reply:
Ok I understand. I think that if only one type of controller is used in the example then it will be easier for the user/learner of ros2 control to follow the intent. After this example another example can be used to showcase the CLI capabilities regarding the loading/unloading of another type of controller. 

We can leave both controllers in the controllers.yml and start with the joint_trajectory_controller and the switching can be done to the forward_position_controller. This will fit well to the name change to "joint_trajectory_position_controller" as suggested in the second comment. What do you think?

Reply:
@destogl updated everything. By default, the forward_position_controller is started.